### DISCUSSION TITLE
Lazy loading chunk with Module Federation

### DISCUSSION THREAD

**Discussion by kmisha** [nrwl/nx#15792] [General]
Hello community!

I'm going to use Nx Module Federation for my current Angular project. But I am getting all my node dependencies packages build as separate lazy loaded chunk files now.

For example:

- without MF I have: main, runtime, polyfills, styles and 3 lazy chunk.
- with MF I have: main, polyfills, styles and over 50 lazy chunk files.

Have you ever meet with this case? How did you solve this problem?

**Comment by kmittal** [comment]
@kmisha : Were you able to find out why the `node_modules` is built as a separate lazy-loaded chunk files? I am seeing similar behavior like you reported in one of my projects.

**Reply by kmisha** [reply]
Unfortunately, I decided to drop out MF from my project.

**Reply by kmittal** [reply]
Oh alright. Thanks for your quick response.
